src/common/meta/src/key.rs

```diff
@@ -226,7 +226,7 @@ lazy_static! {
 
 lazy_static! {
     pub static ref KAFKA_TOPIC_KEY_PATTERN: Regex =
-        Regex::new(&format!("^{KAFKA_TOPIC_KEY_PREFIX}/({NAME_PATTERN})$")).unwrap();
+        Regex::new(&format!("^{KAFKA_TOPIC_KEY_PREFIX}/(.*)$")).unwrap();
```

Comment:
Why change this?

Reply:
This part can be specified by users so that it can be any pattern. In unit test it's a uuid which already does not match `NAME_PATTERN`.